### PR TITLE
[feat] added ter-newline-after-var rule

### PR DESCRIPTION
**Pull request by webschik**
Add [newline-after-var](https://eslint.org/docs/rules/newline-after-var) rule.

### CODE REVIEW

**Comment by webschik** [comment]
Hi, @buzinas !

Could you, please, check this PR?
My team is actively use `tslint-eslint-rules` and I would like to add this rule ;)

File: src/rules/noInvalidRegexpRule.ts [hunk 1]
```diff
@@ -26,9 +26,9 @@ class NoInvalidRegexpWalker extends Lint.RuleWalker {
         const expr = (args[0] as ts.StringLiteral).text;
         const flags = args.length > 1 && args[1].kind === ts.SyntaxKind.StringLiteral ? (args[1] as ts.StringLiteral).text : undefined;
 
-        let regex: RegExp;
         try {
-          regex = new RegExp(expr, flags);
+          // tslint:disable-next-line no-unused-expression
```
Comment (jmlopez-rod) [inline src/rules/noInvalidRegexpRule.ts]: Ah, I guess this would have been a better solution... -_-.

**Comment by jmlopez-rod** [comment]
@webschik Can you resolve the conflict that I introduced? I should have merged your PR first. I like your solution. Just keep your file.

**Comment by webschik** [comment]
Hi @jmlopez-rod !
Sorry for my late reply, I wouldn't touch the `src/rules/noInvalidRegexpRule.ts` file, because I do not use it for the `newline-after-var` rule, but without my "fix" on line 30 I can't build the project and pass the tests.

**Pull request merged**